component-runtime-impl/src/main/java/org/talend/sdk/component/runtime/record/RecordImpl.java

```diff
@@ -128,12 +128,12 @@ public BuilderImpl(final Schema providedSchema) {
             return entry;
         }
 
-        private Schema.Entry findOrBuildEntry(final String name, final Schema.Type type, final boolean nullable) {
+        private Schema.Entry findOrBuildEntry(final String name, final Schema.Type type) {
             if (providedSchema == null) {
                 return new SchemaImpl.EntryImpl.BuilderImpl()
                         .withName(name)
                         .withType(type)
-                        .withNullable(nullable)
+                        .withNullable(true)
```

Comment:
The goal of the PR is to return Objects and not primitives (so we may have null values indeed) but the fact is that WithNullable is a schema parameter that must be honored when setting/getting the record value by the implementation. Hope that I'm clear otherwise ping me on slack.

Reply:
This method is "private" on builder to add field schema from its name.
Nullable false was employed for all primitives types (int, long, boolean ...) as a null value could not be assigned. Now that we have no primitive, null value can be everywhere, so nullable is always true.